packages/react-charts/src/components/ChartAxis/ChartAxis.tsx

```diff
@@ -462,7 +470,14 @@ export const ChartAxis: React.FunctionComponent<ChartAxisProps> = ({
   });
 
   // Note: containerComponent is required for theme
-  return <VictoryAxis containerComponent={container} theme={showGrid ? getAxisTheme(themeColor) : theme} {...rest} />;
+  return (
+    <VictoryAxis
+      containerComponent={container}
+      theme={showGrid ? getAxisTheme(themeColor) : theme}
+      tickLabelComponent={<VictoryLabel id={axisLabelId} />}
```

Comment:
We would want to use our own ChartLabel here. 

We also need to clone the given `tickLabelComponent` here, so we don't override a devs component properties.